node/rpc/data_worker_ipc_server.go

```diff
@@ -152,10 +153,7 @@ func (r *DataWorkerIPCServer) CalculateChallengeProof(
 	}
 
 	if !found {
-		return nil, errors.Wrap(
```

Comment:
Wrapper errors are unfortunately just propagated as strings via gRPC, so the original structure is lost. Status codes however are the standard way of communicating errors.